authentication-service/src/main/kotlin/com/saveourtool/save/authservice/utils/SaveUserDetails.kt

```diff
@@ -15,18 +16,21 @@ import org.springframework.security.web.authentication.preauth.PreAuthenticatedA
  * @property id [com.saveourtool.save.entities.User.id]
  * @property name [com.saveourtool.save.entities.User.name]
  * @property role [com.saveourtool.save.entities.User.role]
+ * @property status [com.saveourtool.save.entities.User.status]
  * @property token [com.saveourtool.save.entities.User.password]
  */
 class SaveUserDetails(
     val id: Long,
     val name: String,
     val role: String,
+    val status: String,
```

Comment:
why String and not `UserStatus`?

Reply:
Do not want to `find` required `UserStatus` when parsing headers

Reply:
yeap, you're right. forget about headers